changelogs/fragments/3701-hpilo_info_add_some_functionalities.yml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - hpilo_info - add other return values from HP ILO
```

```suggestion
  - hpilo_info - add return values ``server_name``, ... from HP ILO
```
(list all of them here)